templates/base_index.html

```diff
@@ -27,6 +27,1012 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.lts.short_version }} LTS is her
+                width="131",
+                height="27",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+          <li class="p-inline-images__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/1ac0e434-Nvidia_Logo_Horizontal.svg",
+                alt="Nvidia logo",
+                width="152",
+                height="29",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+          <li class="p-inline-images__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/4dbc1a2d-amd_logo.svg",
+                alt="AMD logo",
+                width="132",
+                height="31",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+          <li class="p-inline-images__item">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/c0ee6182-dell_logo.svg",
+                alt="Dell logo",
+                width="90",
+                height="90",
+                hi_def=True,
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="p-strip--light">
+      <div class="u-fixed-width">
+        <h2>Data centre automation</h2>
+        <p class="p-heading--five">Turn a rack into a bare metal cloud. <a href="https://maas.io/" class="p-link--external">Metal-as-a-Service provisioning</a> of Windows, ESX, Linux.</p>
+        <ul class="p-list is-split">
+          <li class="p-list__item is-ticked">Bare metal cloud with on-demand servers</li>
+          <li class="p-list__item is-ticked">Remote edge cluster operations</li>
+          <li class="p-list__item is-ticked">Infrastructure monitoring and discovery</li>
+          <li class="p-list__item is-ticked">Ansible, Chef, Puppet, SALT, Juju integration</li>
+          <li class="p-list__item is-ticked">Super fast install from scratch</li>
+          <li class="p-list__item is-ticked">VMWare ESX, Windows, CentOS, RHEL, Ubuntu</li>
+          <li class="p-list__item is-ticked">Custom images with pre-installed apps</li>
+          <li class="p-list__item is-ticked">Disk and network configuration</li>
+          <li class="p-list__item is-ticked">API-driven DHCP, DNS, PXE, IPAM, provisioning</li>
+          <li class="p-list__item is-ticked">LDAP authentication and RBAC</li>
+        </ul>
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item"><a href="https://maas.io/" class="p-link--external">Learn more</a></li>
+          <li class="p-inline-list__item"><a href="/contact-us" class="p-link--external js-invoke-modal">Contact us</a></li>
```

Comment:
MAAS has its own contact form https://maas.io/contact-us but if you dont want to use it you should remove the external class and add the chevron.